client/src/components/Input/NotifyPreferenceInput.js

```diff
@@ -80,7 +81,7 @@ const NotifyPreferenceInput = ({
       <WhiteSpace />
       <HelpWrapper>
         {Object.entries(NotifyGroup).map(([key1, label1]) => (
-          <>
+          <div key={key1}>
```

Comment:
What is this code doing? Also the checkbox for new applicant should only be visible to org owner or admin.

Reply:
1. I found that under the iteration of this collection, line83 does not have the key, so there is an error which occurs saying need key, therefore I add key right there. #
2. About the checkbox, actually I found there are two files "EditNotification.js" and "EditOrganisationNotification.js" have almost the same code, what is the difference between both of them? How do I check the user is (org owner or admin) or not?